indi-qhy/qhy_ccd.cpp

```diff
@@ -327,6 +327,13 @@ bool QHYCCD::initProperties()
     IUFillSwitchVector(&CoolerModeSP, CoolerModeS, 2, getDeviceName(), "CCD_COOLER_MODE", "Cooler Mode", MAIN_CONTROL_TAB,
                        IP_RO, ISR_1OFMANY, 0, IPS_IDLE);
 
+
+    //NEW CODE - Add support for overscan/calibration area 
+    IUFillSwitch(&OverscanAreaS[0], "INDI_ENABLED", "Include", (IgnoreOverscanArea ? ISS_OFF : ISS_ON));
+    IUFillSwitch(&OverscanAreaS[1], "INDI_DISABLED", "Ignore", (IgnoreOverscanArea ? ISS_ON : ISS_OFF));
```

Comment:
IgnoreOverscanArea is always true by the time we reach this code. So just set the first switch to ISS_OFF and second switch to ISS_ON

Reply:
Yes, it's also updated in the last commit.